.github/workflows/test-all-custom-flags.yml

```diff
@@ -187,8 +187,46 @@ jobs:
             ./tests-out/videos
             ./tests-out/screenshots
 
-      # Only post back to the PR if there was a failure since test-all has the
-      # typical config case covered for success feedback
```

Comment:
Since this can fail when test-all can pass (and vice-versa), this lack of a success message can be confusing